learnosity_sdk/request/init.py

```diff
@@ -45,6 +47,12 @@ def __init__(
 
         self.security['signature'] = self.generate_signature()
 
+        """
+        We use telemetry to enable better support and feature planning. It is
+        however not advised to disable it, and it will not interfere with any usage.
+        """
```

Comment:
I'd suggest moving this comment down to where we have the enable/disable methods as it's more likely someone will search for those than the private variable. (Just to increase visibility)

Reply:
Cool 👍